osler/inventory/views.py

```diff
@@ -171,14 +178,60 @@ def export_csv(request):
                  drug.expiration_date,
                  drug.lot_number,
                  drug.category,
-                 drug.manufacturer])
+                 drug.manufacturer,
+                 dispensed_sum
+                 ])
         file.seek(0)
         csvfileread = file.read()
 
-    csv_filename = ''.join(['drug-inventory-',str(date.today()),'.csv'])
+    csv_filename = ''.join(['drug-inventory-', str(datetime.date.today()), '.csv'])
 
     response = HttpResponse(csvfileread, 'application/csv')
     response["Content-Disposition"] = (
         "attachment; filename=%s" % (csv_filename,))
 
     return response
+
+@active_permission_required('inventory.export_csv', raise_exception=True)
+def export_dispensing_history(request):
+    start_date = request.POST.get('start_date')
+    end_date = request.POST.get('end_date')
```

Comment:
I'm guilty of this too, and need to eventually change my code, but we should start thinking about how to use forms.py to handle the POST and GET requests. Ideally, all the fields that you want to generate in the html should be in forms.py, and the html should be used to render the fields and GET the inputs back. You probably followed what I did, which works, but I think we should start thinking about using forms.py. If you can implement those changes this pull request, that would be great, but if not, we can work on that later as well.